api/api_suite_test.go

```diff
@@ -10,20 +10,21 @@ import (
 
 	"github.com/DataDog/chaos-controller/api/v1beta1"
 	"github.com/DataDog/chaos-controller/ddmark"
-	"github.com/DataDog/chaos-controller/types"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var client ddmark.Client
+
 func TestApi(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Api Suite")
 }
 
 var _ = BeforeSuite(func() {
-	ddmark.InitLibrary(v1beta1.EmbeddedChaosAPI, types.DDMarkChaoslibPrefix)
+	client, _ = ddmark.NewClient(v1beta1.EmbeddedChaosAPI)
```

Comment:
could we rename this var to ddMarkClient maybe? To be clear about what it is

Reply:
Good idea 👍